ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -8,22 +8,198 @@ import StatusQ.Core.Theme 0.1
 import SortFilterProxyModel 0.2
 import AppLayouts.Profile.stores 1.0 as ProfileStores
 import AppLayouts.Wallet.stores 1.0 as WalletStore
+import AppLayouts.stores.Messaging 1.0 as MessagingStores
 
+// WIP: Previous reorganization step before refactoring `RootStore`
 QtObject {
     id: root
 
-    property var mainModuleInst: mainModule
-    property var walletSectionInst: walletSection
-    property var aboutModuleInst: aboutModule
-    property var communitiesModuleInst: communitiesModule
+    // Global properties that have to remain on `RootStore` (the module instances must be private properties and just used to initialize the
+    // rest and specific stores
+    readonly property bool isProduction: production
+    readonly property bool isOnline: internal.mainModuleInst.isOnline
+    readonly property var sectionsModel: internal.mainModuleInst.sectionsModel
+    readonly property bool sectionsLoaded: internal.mainModuleInst && internal.mainModuleInst.sectionsLoaded
+    readonly property var activeSectionId: internal.mainModuleInst.activeSection.id
+    readonly property var activeSectionType: internal.mainModuleInst.activeSection.sectionType
+
+    // Here define the needed properties that access to `Context Properties`:
+    readonly property QtObject _internal: QtObject{
+        id: internal // Rename to `d` when cleanup done
+
+        readonly property var mainModuleInst: mainModule
+    }
+
+    // Here there should be all the ContextSpecificRootStore objects creation
+    readonly property MessagingStores.MessagingRootStore messagingRootStore: MessagingStores.MessagingRootStore {}
+    readonly property ProfileStores.ProfileSectionStore profileSectionStore: ProfileStores.ProfileSectionStore {}
+    readonly property ContactsStore contactsStore: ContactsStore {}
+
+    // readonly property ChatStores.RootStore rootChatStore: ChatStores.RootStore { ... }
+    // readonly property ActivityCenterStore activityCenterStore: ActivityCenterStore { ... }
+    // readonly property SharedStores.NetworkConnectionStore networkConnectionStore: SharedStores.NetworkConnectionStore { ... }
+    // + all the rest of stores now created on `AppMain`
```

Comment:
Commented out code?

Reply:
No, it's just a guide for next refactoring iteration.